worker/src/index.ts

```diff
@@ -118,11 +124,19 @@ class BrowserWorker {
 
             await this.listenForCommands();
 
-            this.browserServer = await chromium.launchServer({
-                port: this.config.server.port,
-                headless: this.config.server.headless,
-                wsPath: `/playwright/${this.workerId}`,
-            });
+            if (this.config.server.browserType === 'chromium') {
```

Comment:
Both branches duplicate the same options, which risks drift, and the current logic effectively makes Firefox the default (anything not 'chromium').

I would suggest picking the browser and launching once, for example:
`this.browserServer = await browsers[browserType].launchServer(...)`